app/store/db.go

```diff
@@ -19,25 +19,48 @@ type DataStore struct {
 }
 
 var (
-	dbDir    string
-	dbFile   string
-	saltFile string
+	dbDir          string
+	dbFile         string
+	saltFile       string
+	sessionsSchema = `
+	CREATE TABLE IF NOT EXISTS sessions (
+		id INTEGER PRIMARY KEY, 
+		name text, 
+		tel text, 
+		isgroup boolean, 
```

Comment:
There is a mixture in the following lines between the format strStr and strstr; i.e. _isgroup_ and _expireTimer_. If it is intended, ignore this comment.